indy_common/test/types/test_get_auth_rule_schema.py

```diff
@@ -22,3 +22,11 @@ def test_has_expected_validators():
     schema = dict(ClientGetAuthRuleOperation.schema)
     for field, validator in EXPECTED_ORDERED_FIELDS.items():
         assert isinstance(schema[field], validator)
+
+
```

Comment:
I think we need to mention `AuthRuleValueField` in `EXPECTED_ORDERED_FIELDS ` above as a value for `"old_value"` and `"new_value"`